torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -1134,9 +1134,9 @@ struct GraphFuser {
     replaceIntermediateBroadcastingChunks();
 
     // Fuse starting chunks into the group.
-    for (auto it = block_->nodes().rbegin(); it != block_->nodes().rend();) {
-      it = scanNodeForChunks(*it);
-    }
+    //for (auto it = block_->nodes().rbegin(); it != block_->nodes().rend();) {
+     // it = scanNodeForChunks(*it);
+    //}
```

Comment:
This change is probably not needed?

Reply:
You are right, it is not supposed to be in this PR.